BUILDING.md

```diff
@@ -0,0 +1,87 @@
+# Build-Steps
+
+The J2V8 build-system performs several build steps in a fixed order to produce the final J2V8 packages for usage on the designated target platforms. What follows is a short summary for what each of the executed build-steps does and what output artifacts are produced by each step.
+
+---
+## Node.js
+
+Builds the [Node.js](https://nodejs.org/en/) & [V8](https://developers.google.com/v8/) dependency artifacts that are later linked against by the J2V8 native bridge code.
+
+__Inputs:__
+- Node.js source code
+    - see [Github](https://github.com/nodejs/node)
+- Node.js GIT patches with customizations for integrating Node.js into J2V8
```

Comment:
This is more or less there for convenience / to be used if necessary ... I know there has been work in other issues / PRs to get the build working without any customization to node (which would be preferred of course), but until we get there, this was a clean way that I would use to overcome any such issues in the meantime.